compiler-plugin-tests/src/test/resources/ballerina_sources/sample_codeaction_package_2/service.bal

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+service /greeting on new http:Listener(0) {
+    resource function get hi () returns string {
```

Comment:
```suggestion
    resource function get hi() returns string {
```

Reply:
Let's remove other spaces as well

Reply:
Fixed in [`239e4bd`](https://github.com/ballerina-platform/module-ballerina-http/pull/880/commits/239e4bdc40cf610cabdae97329c50c88c8a30f59)